src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -266,18 +266,16 @@ private void setScreenMode(Configuration config) {
     }
   }
 
+  // This is only called when internetAccess == true or for mailto/openpgp4fpr scheme,
+  // because when internetAccess == false, the page is loaded inside an iframe,
+  // and WebViewClient.shouldOverrideUrlLoading is not called for HTTP(S) links inside the iframe
```

Comment:
```suggestion
  // This is usually only called when internetAccess == true or for mailto/openpgp4fpr scheme,
  // because when internetAccess == false, the page is loaded inside an iframe,
  // and WebViewClient.shouldOverrideUrlLoading is not called for HTTP(S) links inside the iframe
```